lib/index.js

```diff
@@ -45,7 +45,14 @@
 
         // The user's cookies are required by the API to properly handle the request.
         // Keep a reference so that we can ensure we use updated cookies.
-        this._cookies = options.cookies || {};
+        this._cookies = {};
+        if (options.cookies) {
+            var cookies = options.cookies.split(/; */);
```

Comment:
Can we initialize this regex earlier so it's not being recompiled each time this hits?